core/templates/dev/head/editor/exploration_editor.html

```diff
@@ -91,34 +91,34 @@
     {% if username %}
       <li id="tutorialPreviewTab" ng-class="{'active': getTabStatuses().active === 'preview'}">
         <a href="#" tooltip="Preview" tooltip-placement="bottom" ng-click="selectPreviewTab()" class="protractor-test-preview-tab">
-          <span class="glyphicon glyphicon-play"></span>
+          <i class="material-icons">&#xE037;</i>
```

Comment:
Yup I have made the changes in oppia.css. Is it the wrong place to change that? @seanlip